system/lib/libc/musl/arch/emscripten/bits/fenv.h

```diff
@@ -1,11 +1,4 @@
-#define FE_INVALID    1
-#define __FE_DENORM   2
-#define FE_DIVBYZERO  4
-#define FE_OVERFLOW   8
-#define FE_UNDERFLOW  16
-#define FE_INEXACT    32
```

Comment:
Are these designed to be disabled by simply not being defined?  That seems unusual, but I don't know this part of the standard.

Reply:
Yeah, I learned this today as well. It is quite unusual for the C spec, I agree.

>  Libraries may define in <fenv.h> only the macro values above they support (the others may not be defined).

http://www.cplusplus.com/reference/cfenv/FE_INEXACT/

And musl in fact checks whether things are defined in order to ifdef relevant code. Which actually is helpful in this case!

Reply:
Not sure if they are designed that way exactly, but  musl code seems to use them that way: see e.g. https://github.com/emscripten-core/emscripten/blob/master/system/lib/libc/musl/src/fenv/__flt_rounds.c and in particular https://github.com/emscripten-core/emscripten/blob/2bca083cbbd5a4133db61fbd74d04f7feecfa907/system/lib/libc/musl/src/math/lrint.c gates its use of the FENV_ACCESS pragma on `FE_INEXACT` being defined.